DashlistPanel: Add options to include time range and variable values (#65757)
grafana/grafana: docs/sources/panels-visualizations/visualizations/dashboard-list/index.md

Use these options to refine your visualization.

- **Time range -** Select this option to propagate the time range of the current dashboard to the links. When the user clicks a link, the linked dashboard opens with the inducated time range already set.

> - **Variable values -** Select this option to include template variables currently used as query parameters in the link. When the user clicks the link, any matching templates in the linked dashboard are set to the values from the link. For more information, see [Dashboard URL variables]({{< relref "../../../dashboards/build-dashboards/create-dashboard-url-variables/" >}}).

Reviewer: ```suggestion
- **Include current template variable values -** Select this option to include template variables currently used as query parameters in a link. When the user clicks the link, any matching templates in the linked dashboard are set to the values from the link. Learn more about [Dashboard URL variables]({{< relref "../../../dashboards/build-dashboards/create-dashboard-url-variables/" >}}).
```
